changelog/2023-09-10-addedInsertPythonAction.md

```diff
@@ -0,0 +1,7 @@
+---
+tags: [enhancement, talon]
+mergeDate: 2023-09-10
```

Comment:
I thought we were going to have a bot fix these up. But no harm in having here now in the interim.

Reply:
Yeah eventually, but for now seemed better to at least have them